test/cmdln_parser_test.py

```diff
@@ -76,18 +73,19 @@ def testFormatOption(self):
         for format in self.formats:
             sys.argv = ['scout', '--format', format, 'python']
 
-            ret = self.parser.parse_args()
-            self.assertEqual(self.parser.format, format)
+            self.parser.parse_args()
+            self.assertEqual(self.parser._format, format)
 
-# This test doesn't works, because OptionParser calls a sys.exit()
-# and that's not catchable thing in Python :-(
-# FIXME: a redefine of the parse_args method in OptionParser subclass but this
-#        is not really interesting for normal use, only for testing
+# This test doesn't works, because OptionParser calls a sys.exit() and
```

Comment:
Don't comment out, add ``unittest.skip('Not implemented')`` or something.

Reply:
I did not commented the original test, only fixed the comment itself.